gensim/topic_coherence/indirect_confirmation_measure.py

```diff
@@ -26,6 +26,23 @@
 
 logger = logging.getLogger(__name__)
 
+
+def _present(w_prime_star, w, w_backtrack):
+    """
+    Internal helper function to return index of (w_prime_star, w) in w_backtrack.
+    Return -1 if not present.
+    """
+    index = -1
```

Comment:
yes but I have to maintain `w_prime_star` and `w` tuple in a list which is `w_backtrack` since numpy arrays are unhashable. So this is basically a lookup of a numpy tuple inside the backtracking list. What could be a better way to do this?